activity_browser/app/ui/tables/inventory.py

```diff
@@ -124,8 +124,8 @@ def __init__(self):
         self.database_name = None
         self.dataframe = pd.DataFrame()
         self.technosphere = True
-        self.act_fields = lambda: [f for f in ['reference product', 'name', 'location', 'unit'] if f in AB_metadata.dataframe.columns]
-        self.ef_fields = lambda: [f for f in ['name', 'categories', 'type', 'unit'] if f in AB_metadata.dataframe.columns]
+        self.act_fields = lambda: AB_metadata.get_existing_fields(['reference product', 'name', 'location', 'unit'])
```

Comment:
Does this still need to be a lambda? Check and update if not.

Reply:
Yes, lambda needs to stay, act_fields and ef_fields are called as functions in the sync function further down.